app/controllers/ViewsController.scala

```diff
@@ -12,9 +12,15 @@ class ViewsController(val acl: Acl, assetsManager: AssetsManager, isDev: Boolean
                       crypto: Encryption, val deps: BaseFaciaControllerComponents)(implicit ec: ExecutionContext) extends BaseFaciaController(deps) {
 
   def priorities() = AccessAuthAction { request =>
-    val identity = request.user
-    Cached(60) {
-      Ok(views.html.priority(Option(identity), config.facia.stage, isDev, true))
+    val shouldRedirect: Boolean = request.queryString.getOrElse("redirect", Seq("true")).contains("true")
```

Comment:
If, for whatever reason` we needed to go back to the index page of V1, then `?redirect=false` will do it. (Or really, value, other than `true`).